devops/ansible/roles/large_image/tasks/pil.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install pil tile source system dependencies
```

Comment:
Let's move all of this file's content up to the `tasks/main.yml` file. Accordingly, Pillow will always be installed with `large_image`, since we need it to do post-processing / transforms on other TileSource types.

Since the `pil` TileSource no longer has distinct requirements, let's remove this file and all references to `pil` for `large_image_tile_sources`.